src/styles/Theme.ts

```diff
@@ -1,27 +1,30 @@
 import { DefaultTheme } from 'styled-components'
 
-export const theme: DefaultTheme = {
+export const defaultTheme: DefaultTheme = {
     colors: {
-        primary: '#FAFAFA',
-        secondary: '#FFC80A',
-        tertiary: '#303030',
-        background: '#ffffff',
+        primary: '#0075BA',
+        secondary: '#80BADD',
+        tertiary: '#C0DDEE',
+        background: '#FFFFFF',
         text: '#000000',
-        primaryTextColor: '#594F43',
+        primaryTextColor: '#0085FF',
         secondaryTextColor: '#777777',
         inputPlaceholder: '#C7C7C7',
         darkGrayText: '#303030',
         darkText: '#1A1A1A',
-        black: '#000000',
-        white: '#ffffff',
-        dark: '',
-        medium: '',
-        light: '',
-        danger: '',
-        success: '#66A15A',
+        danger: '#F32013',
+        success: '#5CB85C',
     },
     fonts: {
-        anekMalayalam: 'Anek Malayalam',
+        fontFamily: 'Anek Malayalam',
+        fontSize: {
+            xs: '8px',
```

Comment:
I've watched the video about **em** and **rem** (https://youtu.be/_-aDOAMmDHI). I think It might be really convenient to use rem instead of **px**. 
`xs: '0.5rem', sm: '1rem', md: '1.5rem', lg: '2rem', xl: '2.5rem', xxl: '3rem',`